src/app/profile/core.nim

```diff
@@ -53,6 +53,7 @@ proc init*(self: ProfileController, account: Account) =
   self.view.setNewProfile(profile)
   self.view.network.setNetwork(network)
   self.view.ens.init()
+  self.view.initialized()
```

Comment:
Preferred to have this in `loginModel` but that one is destroyed by the time we've logged-in